src/crawlee/project_template/{{cookiecutter.project_name}}/{{cookiecutter.__package_name}}/__main__.py

```diff
@@ -0,0 +1,6 @@
+import asyncio
+
+from .main import main
+
+if __name__ == '__main__':
+    asyncio.run(main())
```

Comment:
since in this file it is not problematic, I would format it correctly
```suggestion
import asyncio

from .main import main


if __name__ == '__main__':
    asyncio.run(main())
```